### DISCUSSION TITLE
We'd love your feedback on the Trending tab!

### DISCUSSION THREAD

**Discussion by eggimo** [community/community#32946] [Other Feature Feedback, Questions, & Ideas]
Hello everyone! I'm a designer working on the [Trending tab](http://github.com/trending). The team is working on improving the experience and would love to get feedback from folks who use it. 

This is an unmoderated UserTesting session that will be recorded. The session will take about ~15 minutes of your time, and you will be compensated with $30.

If you're interested in participating, please send an email to eggimo@github.com, and I will follow up with the test details.

Thank you for helping us make GitHub better!

**Comment by radonn24** [comment]
congratulations ... marking this comment as "disruptive content" is LITERALLY proving my point

----

`git branch -D featured`

`git branch | Select-String -NotMatch -Pattern "master" | %{ git branch -D $_.ToString().Trim() }`

![image](https://user-images.githubusercontent.com/13595375/190214597-8a821df7-5c53-4012-9554-baa88730dae9.png)

![image](https://user-images.githubusercontent.com/13595375/190208519-7b9ef4a4-8a14-4bef-9ceb-a31b343e7a91.png)
